docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,187 @@
+* Run a Prometheus configuration natively using Grafana Agent Flow
+
+[Prometheus]: https://prometheus.io/docs/prometheus/latest/configuration/configuration/
+
+## Components used in this topic
+
+* [prometheus.scrape][]
+* [prometheus.remote_write][]
+
+[prometheus.scrape]: {{< relref "../reference/components/prometheus.scrape.md" >}}
+[prometheus.remote_write]: {{< relref "../reference/components/prometheus.remote_write.md" >}}
+
+## Before you begin
+
+* Have an existing Prometheus configuration.
+* Have a set of Prometheus applications ready to push telemetry data to
+  Grafana Agent Flow.
+* Be familiar with the concept of [Components][] in Grafana Agent Flow.
+
+[Components]: {{< relref "../concepts/components.md" >}}
+[convert]: {{< relref "../reference/cli/convert.md" >}}
+[run]: {{< relref "../reference/cli/run.md" >}}
+[Flow Debugging]: {{< relref "../monitoring/debugging.md" >}}
+[debugging]: #debugging
+
+## Convert a Prometheus configuration
+
+In order to fully migrate your configuration from [Prometheus] to Grafana Agent
+Flow, your Prometheus configuration must be converted into a Grafana Agent Flow
+configuration. This will enable you to modify it going forward as a flow
+configuration and take full advantage of the many additional features available
+in Grafana Agent Flow.
+
+> In this task, we will use the [convert][] CLI command to output flow
+> configuration from a Prometheus configuration.
+
+1. Execute the following
+
+```bash
+grafana-agent convert --format=prometheus --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+  
+> Replace `INPUT_CONFIG_PATH` with the full path to the Prometheus configuration.
+>
+> Replace `OUTPUT_CONFIG_PATH` with the full path to output the flow configuration.
+
+The following example demonstrates converting a Prometheus configuration:
+
+```yaml
+global:
+  scrape_timeout:    45s
+
+scrape_configs:
+  - job_name: "prometheus"
+    static_configs:
+      - targets: ["localhost:12345"]
+
+remote_write:
+  - name: "grafana-cloud"
+    url: "https://prometheus-us-central1.grafana.net/api/prom/push"
+    basic_auth:
+      username: USERNAME
+      password: PASSWORD
+```
+
+Execute:
+
+```bash
+grafana-agent convert --format=prometheus --output=OUTPUT_CONFIG_PATH INPUT_CONFIG_PATH
+```
+
+The contents of the new flow configuration:
+
+```river
+prometheus.scrape "prometheus" {
+	targets = [{
+		__address__ = "localhost:12345",
+	}]
+	forward_to     = [prometheus.remote_write.default.receiver]
+	job_name       = "prometheus"
+	scrape_timeout = "45s"
+}
+
+prometheus.remote_write "default" {
+	endpoint {
+		name = "grafana-cloud"
+		url  = "https://prometheus-us-central1.grafana.net/api/prom/push"
+
+		basic_auth {
+			username = "USERNAME"
+			password = "PASSWORD"
+		}
+
+		queue_config {
+			capacity             = 2500
+			max_shards           = 200
+			max_samples_per_send = 500
+		}
+
+		metadata_config {
+			max_samples_per_send = 500
+		}
+	}
+}
+```
```

Comment:
Hm, while I like the example I'm not sure if we should include examples in the guide topic. 

WDYT @clayton-cornell

Reply:
I think we do examples in the other getting-started docs, but I'm good with whatever we decide is correct for task-based docs.

Reply:
Examples are useful in making things very clear and gives readers a chance to see what happens in a particular step. I'd leave it for now... but it does need some tweaking,